### PR TITLE
Add tabs to Snippets

### PR DESCRIPTION
**Pull request by danrot**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | ---
| Related issues/PRs | ---
| License | MIT
| Documentation PR | ---

#### What's in this PR?

This PR adds tabs to the snippet form.

#### Why?

Because we will have multiple tabs of content on forms in the future (think SEO and Taxonomies Tab).

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/variables.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+$viewPadding: 60px;
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/variables.scss]: I am not sure about this... I think it is a nice idea, to keep the padding consistent. I am only not sure if this should be part of another SCSS file, or if we should introduce a `variables.scss` as I have done here.

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/toolbar.scss [hunk 2]
```diff
@@ -7,8 +7,10 @@ $itemsContainerBorderColor: $gallery;
 
 $itemsContainerBorder: 1px $itemsContainerBorderColor solid;
 
+$toolbarHeight: 60px;
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/toolbar.scss]: Isn't really consistent with the `variables.scss` file above... But the above variable isn't used in its own bundle.

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/variables.scss (hunk 1 above)
Comment (chirimoya) [reply src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/variables.scss]: ack

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/toolbar.scss (hunk 2 above)
Comment (chirimoya) [reply src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/toolbar.scss]: ack

File: src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/tests/registries/RouteRegistry.test.js [hunk 3]
```diff
@@ -69,8 +85,26 @@ test('Add a route collection to the RouteRegistry', () => {
...
+        parameters: {
+            test: 'value',
+        },
+        parent: undefined,
+        children: [],
+    });
+    expect(routeRegistry.get('route2')).toEqual({
+        name: 'route2',
+        view: 'view2',
+        pattern: '/route/2',
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/tests/registries/RouteRegistry.test.js]: path

**Pull request merged**